routes/index.js

```diff
@@ -14,4 +18,31 @@ router.get('/test-db', function(req,res) {
     })
 })
 
+// Route for login/adding new user
+router.post('/api/users', function(req,res){
+    const { firstname, lastname, email } = req.body
+    const secret = uuidv1();
+
+    if (!firstname || !lastname || !email) {
+        res.status(400).send('Input field empty');
+        alert('Input field empty');
+      } else {
```

Comment:
[nit] If you add a return in front of `res.status(400).send('Input field empty');`, then you can remove the `else` block. It looks a bit nicer to me, since its part of the "happy path" - code that's intended to be run (the if is just validation).